qanta/wikipedia/cached_wikipedia.py

```diff
@@ -149,7 +151,28 @@ def load_page(self, key: str):
             return CachedWikipedia.load_page(key)
         return raw
 
+    def get_wikidata(self, key: str):
+        web_page = "https://www.wikidata.org/w/api.php?action=wbsearchentities&search=" + key + "&language=en&format=json"
+        headers = {"Accept": "application/json"}
+        req = requests.get(web_page, headers=headers)
+
+        try:
+            if not req.json()['search']:
+                print('No results ' + search_term + " : " + search_term)
```

Comment:
Instead of print statements use logging like here https://github.com/Pinafore/qb/blob/master/qanta/extract_features.py#L28